### DISCUSSION TITLE
/v2/zh/config/frontmatter/layout.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#228] [Announcements]
# /v2/zh/config/frontmatter/layout.html

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/config/frontmatter/layout.html

**Comment by rockbenben** [comment]
toc 区块页面位置异常，跑到右边去了。

另外问下，toc 能不能指定哪个标题不显示在目录栏？

**Comment by Mister-Hope** [comment]
你说的异常，是什么异常呢

**Reply by rockbenben** [reply]
https://vuepress-theme-hope.github.io/v2/zh/config/frontmatter/layout.html#toc

toc 标题跑到目录栏的右侧了

**Reply by Mister-Hope** [reply]
所以在你眼里，这些网站都是有问题的是吧？嗯？

- https://www.typescriptlang.org/docs/handbook/2/basic-types.html

- https://babeljs.io/docs/en/babel-preset-env

- https://developers.weixin.qq.com/miniprogram/dev/devtools/download.html

- https://flutter.cn/docs/development/ui/widgets-intro#using-material-components

就，怎么说呢，你也别说我嘲讽你，能抢救一下就抢救一下，不能抢救就这样好吧。不建议非得出来到处气人

**Reply by rockbenben** [reply]
只有我一个人看着有问题吗……
https://s3.bmp.ovh/imgs/2022/07/22/320209c4cacb9486.png

**Reply by rockbenben** [reply]
![](https://s3.bmp.ovh/imgs/2022/07/22/320209c4cacb9486.png)
